wallet-sdk/src/main/kotlin/org/stellar/walletsdk/util/FormatOperation.kt

```diff
@@ -0,0 +1,212 @@
+package org.stellar.walletsdk.util
+
+import org.stellar.sdk.Asset
+import org.stellar.sdk.responses.operations.CreateAccountOperationResponse
+import org.stellar.sdk.responses.operations.OperationResponse
+import org.stellar.sdk.responses.operations.PathPaymentBaseOperationResponse
+import org.stellar.sdk.responses.operations.PaymentOperationResponse
+import org.stellar.walletsdk.*
+
+/**
+ * Format Stellar account operations to make them consistent.
+ *
+ * @param accountAddress Stellar address of the account
+ * @param operation Stellar operation to format
+ *
+ * @return formatted operation
+ */
+fun formatStellarOperation(
+  accountAddress: String,
+  operation: OperationResponse
+): WalletOperation<OperationResponse> {
+  when (operation.type) {
+    // Create account
+    "create_account" -> {
+      val isCreator = (operation as CreateAccountOperationResponse).funder == accountAddress
+
+      return WalletOperation(
```

Comment:
I think there should be WalletOperationBuilder after all...
I notice a lot of places are similar, adding fields or new types of operations can be bothersome. 
I propose a standard builder class (advantage is reusing the code and pre-filling it with defaults)

```kotlin
class WalletOperationBuilder() {
  lateinit var id
  lateinit var createdAt
  lateinit var amount
  ...
}
```
usages:
```kotlin
val builder = builder.fromOperation(operation)
when operation
   is CreateAccountOperationResponse -> builder.amount(wallet.amount)
      .accountFrom(operation) // Move logic to builder
      .account(if (isCreator) operation.account else operation.funder) // Keep logic here
      .nativeAsset().
      .type(<...>)
   else -> builder.defauts()

return builder.build()
```
Can also use dls style (optional)
```kotlin
when operation
   is CreateAccountOperationResponse -> builder.build {
     amount = wallet.amount
     account = if (isCreator) operation.account else operation.funder)
     nativeAsset()
     type = <...>
   }
```

Reply:
Is this pattern preferable to what we currently have in Kotlin? It's harder for me to read it this way, but I won't pretend that I'm a Kotlin developer. :sweat_smile: We can update this in a follow-up PR.

Reply:
I agree. Yes, it's common in Kotlin, as it comes mainly from java world